client/src/components/Home.jsx

```diff
@@ -104,21 +109,15 @@ const Home = ({ setIsLoading, isMobile, isLoggedIn, isLoading, setIsLoggedIn, sh
 
     let incomingId;
 
-    if (info.length > 0) {
-      incomingId = info.calendars[0].calendar_id;
-    } else {
-      incomingId = 11;
-    }
+    incomingId = userCalendar.calendar_id;
 
     axios.post('http://localhost:3000/todoList/category', { params: { calendar_id: incomingId, category: category } })
       .then((result) => {
-        console.log('cat post result: ', result);
         let catId = result.data.category_id;
         let newCat = { category_id: catId, category: category, todoitems: [] };
-        let newEventsList = myEvents[0];
+        let newEventsList = myEvents;
         newEventsList.push(newCat);
```

Comment:
You could also do a getAllTodos call here, rather than pushing into state. But this is fine if it works.

Reply:
I agree, and that would be better, but it works, so don't want to mess with it--as I really need to move to the next task right now.